dev/2017/fall/knights-racing/knights-racing.scss

```diff
@@ -0,0 +1,178 @@
+@import '../../../../versions/v4/static/scss/variables';
+@import '../../../../versions/v4/static/scss/mixins';
+
+$sans-serif: 'Arial Black', arial;
+$sans-serif-cond: 'Open Sans Condensed', arial;
+$story-red: #d6553a;
+$story-white: #fff;
+$story-black: #333;
+$story-gray: #4d4d4d;
+$story-back: #d3d6db;
+
+// Resets
+article {
```

Comment:
Make sure all story styles are prefixed with article.story to avoid overriding header and footer styles.